src/main/java/it/pagopa/interop/probing/eservice/registry/updater/consumer/ServicesReceiver.java

```diff
@@ -87,6 +89,7 @@ public void receiveStringMessage() throws IOException {
         AWSXRay.endSegment();
       }
       sqsMessages = sqs.receiveMessage(receiveMessageRequest).getMessages();
+      logger.logNumberMessagesReceived(Objects.nonNull(sqsMessages) ? sqsMessages.size() : 0);
```

Comment:
may you print two different messages when logging before and after the loop?

Reply:
It is the exact same operation. We just repeat it to iterate over the messages.

Reply:
you're right, I misread the code block 👍